src/herbpy/action/blocks.py

```diff
@@ -62,9 +62,9 @@ def _GrabBlock(robot, blocks, table, manip=None, preshape=None,
             with env:
                 start_point = manip.GetEndEffectorTransform()[0:3, 3]
                 table_aabb = ComputeEnabledAABB(table)
-                table_height = table_aabb.pos()[2] + table_aabb.extents()[2]
+                #table_height = table_aabb.pos()[2] + table_aabb.extents()[2]
                 # 0.14 is the distance from finger tip to end-effector frame
-                current_finger_height = manip.GetEndEffectorTransform()[2,3] - 0.14
+                #current_finger_height = manip.GetEndEffectorTransform()[2, 3] - 0.14
```

Comment:
Well we never used this variable, but I was a bit conservative in that maybe people would want the reference of how to get the current finger height (I assume it was here for a reason at some point). 
I can just as easily go ahead and delete it.